src/librustc_yk_sections/mir_cfg.rs

```diff
@@ -243,40 +249,43 @@ impl<'a, 'tcx> PhiInserter<'a, 'tcx> {
     /// Insert PHI nodes, returning the mutated pack.
     fn pack(mut self) -> ykpack::Pack {
         let mut df = DominatorFrontiers::new(self.mir);
+        let num_tir_vars = self.def_sites.len();
+        let num_tir_blks = self.mir.basic_blocks().len(); // There's one TIR block per MIR block.
 
         // We first need a mapping from block to the variables it defines. Appel calls this
         // `A_{orig}`. We can derive this from our definition sites.
-        let (a_orig, num_blks) = {
+        let a_orig = {
             let ykpack::Pack::Mir(ykpack::Mir{ref blocks, ..}) = self.pack;
-            let mut a_orig: IndexVec<BasicBlock, FxHashSet<TirLocal>> =
-                IndexVec::with_capacity(blocks.len());
-            a_orig.resize(blocks.len(), FxHashSet::default());
+            debug_assert!(num_tir_blks == blocks.len());
```

Comment:
This assert feels like it might be better off at line 252ish?

Reply:
I think that was removed in the last commit, where I actually count the TIR blocks instead of getting the number of blocks from the MIR.